app/graphql/types/user_type.rb

```diff
@@ -13,12 +13,12 @@ def weekly_moods
       object.moods.order(created_at: :desc).limit(7)
     end
 
-    def habits
-      object.habits.distinct
-    end
-
     def journals
       object.journals.distinct
     end
+
+    def habits
+      object.habits.distinct
```

Comment:
Yooo, wtf. I was getting this bug too, any idea why?

Reply:
No clue